osf/utils/migrations.py

```diff
@@ -2,13 +2,32 @@
 import json
 import logging
 
+from contextlib import contextmanager
+
 from website import settings
 from osf.models import NodeLicense, MetaSchema
 from website.project.metadata.schemas import OSF_META_SCHEMAS
 
 logger = logging.getLogger(__file__)
 
 
+@contextmanager
+def disable_auto_now_fields(model):
+    """
+    Context manager to disable updates of all auto_now fields for a given model.
+
+    """
+    for field in model._meta.get_fields():
```

Comment:
It would be nice to optionally pass fields:

```python
with disable_auto_now_fields(SomeModel):  # disable all auto_now fields

with disable_auto_now_fields(SomeModel, ['date_modified']):  # disable specified fields
```